File: src/main/java/com/p2p/lending/dao/DopeDao.java
package com.p2p.lending.dao;

import java.util.List;
import java.util.Map;

import com.p2p.lending.entity.Dope;

public interface DopeDao {
	int insert(Dope dope);
	//分页查询
	public List<Dope> findDope(Map<String, Object> map);
	//查询总行数
	public List total();
	//删除
	public void batchDeletes(Integer did);
}

File: src/main/java/com/p2p/lending/dao/DopeDao.kt
package com.p2p.lending.dao

import com.p2p.lending.entity.Dope

interface DopeDao {
    fun insert(dope: Dope): Int
    //分页查询
    fun findDope(map: Map<String, Any>): List<Dope>

    //查询总行数
    fun total(): List<*>

    //删除
    fun batchDeletes(did: Int?)
}
